src/Configuration/AccountManager.cs

```diff
@@ -0,0 +1,91 @@
+﻿using ClassicUO.Utility;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace ClassicUO.Configuration
+{
+    internal static class AccountManager
+    {
+        public static List<Account> Accounts;
+
+        public static string[] GetAccountNames(string serverName)
```

Comment:
should this include port number?  I'm thinking this is probably overkill but i suppose it is possible that a given server may have different servers running on the same IP and point to different ports.